style.css

```diff
@@ -117,35 +56,92 @@ body {
     letter-spacing: 1px;
 }
 .primary-btn{
-    background-color: var(--color-barcelona-red);
+    background-color:var(--color-barcelona-red);
     color: var(--color-white);
     border-radius: 0.25rem;
     font-size: 1rem;
     text-decoration: none;
     padding: 0.5rem 1rem;
 }
+/* .primary-btn:hover{
+    background-color: var(--color-barcelona-red);
+} */
 .secondary-btn{
     color: var(--color-barcelona-blue);
     border-radius: 0.25rem;
     border: 2px solid var(--color-barcelona-blue);
     font-size: 1rem;
     text-decoration: none;
     padding: 0.5rem 1rem;
+    transition: all 0.25s ease-in-out;
+}
+.secondary-btn:hover{
+    background-color: var(--color-light-blue);
 }
 .image-container{
     margin-left: auto;
     flex: 1;
 }
-img{
+.home-img{
     margin-left: 2rem;
-    width: 400px;
+    width: 425px;
 }
-/* footer{
-    background-color: var(--color-darker-gray);
+
+.outest-grid-container{
+    margin-top: 0.5rem;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+    
+    height: 100vh;
+    grid-gap: 0.5rem;
+    grid-template-areas: 
+        "sidebar main main main main"
+    ;
+}
+.sidebar-container{
+    grid-area: sidebar;
+    box-shadow: 0 0 1px 0px rgb(0 0 0 / 25%);
+    padding-top: 0;
+}
+.sidebar-header{
+    margin: 2rem 2rem 0rem 3rem;
     
 }
-footer i{
-    color: var(--color-white);
-    font-size: 1.4rem;
+.sidebar-list{
+    list-style: none;
+    font-size: 1.125rem;
+    padding: 0;
+    
+}
+.sidebar-link-item{
+    padding: 0.75rem 3rem;
+}
+.sidebar-link-item:hover{
+    background-color: var(--color-primary-complimentary)  ;
+}
 
-} */
+.sidebar-list a{
+    text-decoration: none;
+    color: var(--color-darker-gray);
+}
+.main-container{
+    grid-area: main;
+    box-shadow: 0 0 1px 0px rgb(0 0 0 / 25%);
+    padding: 0rem 2rem 2rem 2rem;
+}
+
+.component-container{}
```

Comment:
Here also, you have defined empty class.

Reply:
yes, those were for my reference. I'll delete it later. Thanks.